From ae68a37ca17f5e4b99c290e8cfe4ea907640e14f Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:49 -0400
Subject: [redhat] kabi: Adding symbol vm_stat (mm/vmstat.c)

Message-id: <91b73c5ea86b529a8c37173f9951f4f0e83b17a6.1410510773.git.poros@redhat.com>
Patchwork-id: 96041
O-Subject: [PATCH RHEL7.1 229/233] kabi: Adding symbol vm_stat (mm/vmstat.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Accessed via nr_free_pages. GAB displays this as RCA when it.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/vm_stat
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/vm_stat
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/vm_stat

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/vm_stat  b/redhat/kabi/kabi-module/kabi_ppc64/vm_stat 
new file mode 100644
index 0000000..27fc7d3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/vm_stat  
@@ -0,0 +1,2 @@
+#1-
+0x31a78843 vm_stat vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/vm_stat  b/redhat/kabi/kabi-module/kabi_s390x/vm_stat 
new file mode 100644
index 0000000..ccb5726
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/vm_stat  
@@ -0,0 +1,2 @@
+#1-
+0x14ccd79f vm_stat vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/vm_stat  b/redhat/kabi/kabi-module/kabi_x86_64/vm_stat 
new file mode 100644
index 0000000..27fc7d3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/vm_stat  
@@ -0,0 +1,2 @@
+#1-
+0x31a78843 vm_stat vmlinux EXPORT_SYMBOL
-- 
1.7.1